ci/lint/06_script.sh

```diff
@@ -8,21 +8,19 @@ export LC_ALL=C
 
 set -ex
 
-if [ -n "$LOCAL_BRANCH" ]; then
-  # To faithfully recreate CI linting locally, specify all commits on the current
-  # branch.
-  COMMIT_RANGE="$(git merge-base HEAD master)..HEAD"
-elif [ -n "$CIRRUS_PR" ]; then
+if [ -n "$CIRRUS_PR" ]; then
```

Comment:
Maybe explain why want no more than 2 commits for a PR here. It seems incorrect for `commit-script-check.sh`.

Reply:
Ah, a useful comment could be:

```bash
# Cirrus jobs are run on the 'pull/${CIRRUS_PR}/merge' branch
```

And for safety against regressions maybe add:

```bash
if ! git show --summary HEAD | grep -q ^Merge; then
  echo "Error: expected a merge commit"
  exit 1;
fi 
```

Reply:
(could also check for the presence of `HEAD^2`)

Reply:
I mean checking `HEAD^2` could be an alternative for looking for a commit message regex.

Reply:
`git show HEAD^2` works if you're on a merge commit. It shows the second parent. It fails otherwise, because non-merge commits only have one parent: https://gist.github.com/zhangzhhz/77850a5d6d64568b5ea50ffc7b6c57ce

Reply:
(but your solution is more readable)